airflow/dag_processing/manager.py

```diff
@@ -727,7 +740,8 @@ def _add_callback_to_queue(self, request: CallbackRequest):
                 self._file_path_queue = collections.deque(
                     file_path for file_path in self._file_path_queue if file_path != request.full_filepath
                 )
-            self._file_path_queue.appendleft(request.full_filepath)
+            self._add_paths_to_queue([request.full_filepath], True)
+            Stats.incr("dag_processing.other_callback_count")
```

Comment:
Can you please add description of those metrics to https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/logging-monitoring/metrics.html#metrics

Reply:
Done